doc/plugin_server_upstreamca_spire.md

```diff
@@ -0,0 +1,24 @@
+# Server plugin: UpstreamCA "spire"
+
+The `spire` plugin loads root CA credentials from upstream SPIRE server in the same trust domain, using
+them to generate intermediate signing certificates for the server's signing authority.
```

Comment:
This kind of makes it sound like the plugin receives the root key from upstream SPIRE servers, which is not the case. Perhaps something like

> The `spire` plugin uses credentials fetched from the Workload API to call an upstream SPIRE server in the same trust domain, requesting an intermediate signing certificate to use as the server's X.509 signing authority.

would make that clearer?